src/app/containers/Caption/index.jsx

```diff
@@ -1,33 +1,24 @@
 import React from 'react';
-import { node, string } from 'prop-types';
+import { node } from 'prop-types';
 import Caption from '../../components/Figure/Caption';
-import Text from '../../components/Text';
 import { ServiceContext } from '../../components/ServiceContext';
 import VisuallyHiddenText from '../../components/VisuallyHiddenText';
 
-const CaptionWithOffscreenText = ({ children }) => (
+const CaptionContainer = ({ captionValue }) => (
   <ServiceContext.Consumer>
     {({ imageCaptionOffscreenText }) => (
       <Caption>
         {imageCaptionOffscreenText ? (
           <VisuallyHiddenText>{imageCaptionOffscreenText}</VisuallyHiddenText>
         ) : null}
-        {children}
+        {captionValue}
       </Caption>
     )}
   </ServiceContext.Consumer>
 );
 
-CaptionWithOffscreenText.propTypes = {
-  children: node.isRequired,
-};
-
-const CaptionContainer = ({ captionValue }) => (
-  <Text text={captionValue} paragraphOverride={CaptionWithOffscreenText} />
-);
-
 CaptionContainer.propTypes = {
-  captionValue: string.isRequired,
+  captionValue: node.isRequired,
```

Comment:
It would be best to keep this as type `string`. 

I'd suggest changing the prop name from `captionValue` to `text`, to remove repetition & simplify. 
`<Caption captionValue={captionValue} />` becomes:
`<Caption text={text} />`

This pattern can be updated for other components such as `Copyright` in other PRs.